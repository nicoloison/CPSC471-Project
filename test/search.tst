curl localhost/recipes/search.php
curl localhost/recipes/search.php?author_name=lmitchell
curl "localhost/recipes/search.php?author_name=lmitchell&prep_time=40&prep_time_max=60"
curl "localhost/recipes/search.php?author_name=nloison&name=mac"
