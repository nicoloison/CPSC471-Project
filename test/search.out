{"name":"mac and cheese","author_name":"nloison","instructions":"melt cheese and cook noodles","picture":"pics\/mac.png","prep_time":"45.0","portions":"4","rating":null,"description":"melted cheese and cooked macaronis"}{"name":"shepherd's pie","author_name":"lmitchell","instructions":"cook it for like an hour or whatever","picture":"pics\/pie.png","prep_time":"55.0","portions":"8","rating":null,"description":"potatoes and meat and some corn and stuff!"}{"name":"steak dinner","author_name":"adjuric","instructions":"grill em!","picture":"pics\/steak.png","prep_time":"25.0","portions":"4","rating":null,"description":"grilled meat!"}